File: servicio_banco/src/main/java/org/banco/models/dto/TransaccionQR.java
package org.banco.models.dto;

public class TransaccionQR {
    private String idClienteOrigen;
    private String idCuentaDestino;
    private double monto;
    private String tipo; // "retiro" (para pagar) o "depósito"

    public String getIdClienteOrigen() { return idClienteOrigen; }
    public void setIdClienteOrigen(String idClienteOrigen) { this.idClienteOrigen = idClienteOrigen; }
    public String getIdCuentaDestino() { return idCuentaDestino; }
    public void setIdCuentaDestino(String idCuentaDestino) { this.idCuentaDestino = idCuentaDestino; }
    public double getMonto() { return monto; }
    public void setMonto(double monto) { this.monto = monto; }
    public String getTipo() { return tipo; }
    public void setTipo(String tipo) { this.tipo = tipo; }
}

File: Cliente_Movil/app/src/main/java/com/shibasito/cliente/models/TransaccionQR.kt
package com.shibasito.cliente.models

// DTO para enviar una transacción (leída del QR) al banco
data class TransaccionQR(
    val idClienteOrigen: String,
    val idCuentaDestino: String, // Asumimos que el QR nos da esto
    val monto: Double,
    val tipo: String // "depósito" o "retiro"
)